openrpc.json

```diff
@@ -0,0 +1,1454 @@
+        "schema": {
+          "$ref": "#/components/schemas/TransactionSignatureList"
+        }
+      }
+    }
+  ],
+  "components": {
+    "contentDescriptors": {
+      "Asset": {
+        "name": "Asset",
+        "description": "DAS API Asset",
+        "schema": {
+          "$ref": "#/components/schemas/Asset"
+        }
+      },
+      "AssetList": {
+        "name": "AssetList",
+        "description": "A paged an array of assets",
+        "schema": {
+          "$ref": "#/components/schemas/AssetList"
+        }
+      },
+      "assetProof": {
+        "name": "AssetProof",
+        "description": "Array of hashes corresponding to a `node_index`",
+        "schema": {
+          "$ref": "#/components/schemas/AssetProof"
+        }
+      },
+      "assetProofBatch": {
+        "name": "AssetProofBatch",
+        "description": "Array of hashes corresponding to a `node_index`",
+        "schema": {
+          "$ref": "#/components/schemas/AssetProofBatch"
+        }
+      },
+      "GetAssetProof": {
+        "name": "GetAssetProof",
+        "description": "Payload with public key of asset",
+        "required": true,
+        "schema": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Public key of asset in base58 encoding."
+            }
+          },
+          "required": [
+            "id"
+          ]
+        }
+      },
+      "AssetId": {
+        "name": "AssetId",
+        "description": "Public key of asset in base58 encoding",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      "AssetIds": {
+        "name": "AssetIds",
+        "description": "Asset ids in base58",
+        "required": true,
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Asset public key in base58 encoding"
+          }
+        }
+      },
+      "GetAssetProofBatch": {
+        "name": "GetAssetProofBatch",
+        "description": "Payload with public keys of assets",
+        "required": true,
+        "schema": {
+          "type": "object",
+          "properties": {
+            "ids": {
+              "type": "array",
+              "description": "Public keys of asset in base58 encoding."
+            }
+          },
+          "required": [
+            "ids"
+          ]
+        }
+      }
+    },
+    "schemas": {
+      "after": {
+        "name": "after",
+        "description": "Cursor based pagination: After Asset ID",
+        "schema": {
+          "type": "string"
+        }
+      },
+      "before": {
+        "name": "before",
+        "description": "Cursor based pagination: Before Asset ID",
+        "schema": {
+          "type": "string"
+        }
+      },
+      "page": {
+        "name": "page",
+        "description": "Page based pagination (max 50)",
```

Comment:
This one should be marked as deprecated and discouraged from usage. The recommended way to paginate is using the after or cursor fields. Examples should reflect this as well